src/main/java/Duke.java

```diff
@@ -1,10 +1,194 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    static private final String MAKE_DEADLINE = "deadline";
+    static private final String MAKE_TODO = "todo";
+    static private final String MAKE_EVENT = "event";
+    static private final String DUE_DATE_FLAG = "/by ";
+    static private final String EVENT_START_FLAG = "/from ";
+    static private final String EVENT_END_FLAG = "/to ";
+
+    public static boolean isNumeric(String strNum) {
+        if (strNum == null) {
+            return false;
+        }
+        try {
+            double d = Double.parseDouble(strNum);
+        } catch (NumberFormatException nfe) {
+            return false;
+        }
+        return true;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = " _____  _               _____   _\n"
+                + "|  ___|| |     _    _  |  ___| | |  _\n"
+                + "| |    | |    | |  | | | |     | |/ /\n"
+                + "| |___ | |___ | |__| | | |___  |    \\\n"
+                + "|_____||_____||______| |_____| |_| \\_\\\n";
+
+        System.out.println(logo);
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Howdy! I'm Cluck!\n" +
+                "    What can I cluck-a-doodle-do for you?\n");
+        System.out.println("    ____________________________________________________________");
+
+        boolean loop = true;
+        List<Task> toDoList = new ArrayList<>();
```

Comment:
Perhaps a better name could be used for the ```boolean``` name sound more like a boolean. toDoList could also be renamed to be more specific and descriptive of a container of multiple tasks.
```suggestion
        boolean isActive = true;
        List<Task> tasks = new ArrayList<>();
```